packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -168,4 +169,21 @@ export const Query = {
       familyName: profile?.lastName?.value ?? '',
     }
   },
+  region: async (
+    _: any,
+    {
+      input,
+    }: {
+      input: RegionInput
+    },
+    { clients }: Context
+  ) => {
+    const data = await clients.commerce.checkout.region(input)
+
+    if (data?.[0]?.id) {
+      return data[0].id
+    }
+
+    return null
```

Comment:
How about flipping these? I guess the code currently inside the `if` is the most common path right? The "happy path"

Reply:
Fixed